app/views/tag/_tagging.html.erb

```diff
@@ -48,15 +48,22 @@ $(function () {
 <% parent ||= nil %>
 
 <% if current_user && (parent != :profile || (current_user.id == user.id || logged_in_as(['admin']))) && !@preview  %>
-  <a data-toggle="tooltip" title="Add tags/View all" aria-label="Add tags/View all" id="tags-open" class="btn btn-circle btn-circle-sm tags-open" style="float:left;">
-    <i class="fa fa-plus" style="color:#808080;"></i>
-  </a>
+  <% if current_user.first_time_poster && !(current_user.username == @node.author.username || @node.coauthors.exists?(username: current_user.username) || logged_in_as(['admin', 'moderator'])) %>
```

Comment:
Hi @17sushmita, there is a test failing because of this line `@node.coauthors.exists?`, perhaps if a node has no coauthors it returns nil and becomes `nil.exists?` and maybe that's why the test is failing?

<img width="1062" alt="Screenshot 2021-06-10 at 17 52 38" src="https://user-images.githubusercontent.com/7622875/121547640-fac05f00-ca14-11eb-9725-cb1481694249.png">
nil?

Reply:
Oh yeah! I should check if a coauthor exists in AND condition.